tapgarden/planter_test.go

```diff
@@ -1238,6 +1371,11 @@ var testCases = []mintingStoreTestCase{
 		interval: minterInterval,
 		testFunc: testFinalizeBatch,
 	},
+	{
+		name:     "finalize_with_tapscript_tree",
```

Comment:
FWIW if you leave out the `_` here, it'll insert them automatically as it replaces spaces.